workflows/qc/quality-check-standard.wdl

```diff
@@ -93,24 +91,23 @@ workflow quality_check {
     call fqc.fastqc { input: bam=quickcheck.checked_bam, max_retries=max_retries }
     call ngsderive.instrument as ngsderive_instrument { input: bam=quickcheck.checked_bam, max_retries=max_retries }
     call ngsderive.read_length as ngsderive_read_length { input: bam=quickcheck.checked_bam, bai=bam_index, max_retries=max_retries }
-    
     call ngsderive.encoding as ngsderive_encoding { input: ngs_files=[quickcheck.checked_bam], prefix=prefix, max_retries=max_retries }
-    String parsed_encoding = read_string(ngsderive_encoding.inferred_encoding)
+
+    call picard.bam_to_fastq { input: bam=quickcheck.checked_bam, max_retries=max_retries }
+    call fq.fqlint { input: read1=bam_to_fastq.read1, read2=bam_to_fastq.read2, max_retries=max_retries }
```

Comment:
I agree, I see no reason for us to validate that bam+fastq are the same sample. That's a user error. I don't know if we should hold things up to implement an option like this either. Just a thought for the future, particularly as we start running in real-time rather than on historical cohorts (for which we've likely deleted the fastq files anyway).

Reply:
I also don't think that we output the fastq files from RNA-Seq v2. For WGS/WES, I don't think that `msgen` does either. So it would require some changes there as well.

Reply:
Agreed then. An idea to keep in mind, but not worth implementing now.